addons/website_profile/views/website_profile.xml

```diff
@@ -423,33 +423,40 @@
     </template>
 
     <template id="badge_content" name="Badges Page content">
-        <div class="row">
-            <div class="col-12">
-                <h1 class="mt-4 mt-lg-2">Badges</h1>
-                <p class="lead">
-                    Besides gaining reputation with your questions and answers,
-                    you receive badges for being especially helpful.<br class="d-none d-lg-inline-block"/>Badges
-                    appear on your profile page, and your posts.
-                </p>
+        <div id="badges">
```

Comment:
That feels like a weak id for an element.
Could be dangerous if someone else uses the same id.

Maybe prefix with module, something like `website_profile_badges` ?